backend-rust/src/migrations/m0001-initialize.sql

```diff
@@ -828,3 +833,18 @@ CREATE TABLE block_special_transaction_outcomes (
 -- Index allowing for efficiently finding particular type of outcomes for a particular block.
 CREATE INDEX block_special_transaction_outcomes_idx
     ON block_special_transaction_outcomes (block_height, outcome_type, block_outcome_index);
+
+-- Function for generating a table where each row is a bucket.
+-- Used by metrics queries.
+CREATE OR REPLACE FUNCTION date_bin_series(bucket_size interval, starting TIMESTAMPTZ, ending TIMESTAMPTZ)
```

Comment:
You have to make a release on this. We should stop reusing this file now.

Reply:
We have several breacking changes by now in this file, so we need to reindex anyway which is why I keep extending it.
I agree we should start introducing migrations, but until we are ready to release, we might as well extend on this.